.gitignore

```diff
@@ -34,6 +34,14 @@ yarn-error.log*
 *.tsbuildinfo
 next-env.d.ts
 
+.vs/
+
 /cert
 
+/sitefinity-widget-designers/**/*.js
+/sitefinity-widget-designers/**/*.d.ts
```

Comment:
We could possibly make this compile alongside the main repo. And produce a package from the folder like the framework folder.

Reply:
we can, but I prefer not to. Most probably this package will be separated from this repo and moved to azdo, because adminapp will use it to and it does not have anything related to react.js

Reply:
it is here for now to finish up the designers and get initial reviews